base/system/diskpart/lang/pt-PT.rc

```diff
@@ -94,7 +94,7 @@ BEGIN
     IDS_HELP_CMD_DESC_ACTIVE "ACTIVE\t\t- Marca a partição selecionada como ativa.\n"
```

Comment:
I reject all changes made in this PR relating to the "AO" (Orthographic Agreement 1990). Such as all changes to:
base/applications/mscutils/eventvwr/lang/pt-PT.rc
base/applications/notepad/lang/pt-PT.rc
base/applications/rapps/lang/pt-PT.rc
base/applications/regedit/lang/pt-PT.rc
base/applications/wordpad/lang/pt-PT.rc
base/setup/usetup/lang/pt-PT.h
base/setup/usetup/lang/pt-PT.h
Actualizar to Atualizar,
Seleccionar to Selecionar,
detecta to deteta
correctamente to corretamente
actual to atual
and so on.

Tens de respeitar a língua de Camões e Fernando Pessoa. Não aceitamos palavras estilo Aborto ortográfico.

Reply:
some mentioned files is not my responsibility

Reply:
> some mentioned files is not my responsibility

Right, but let's not add more mistakes to it. You need to revert incorrect changes done by you.